holidays/countries/cape_verde.py

```diff
@@ -0,0 +1,96 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.constants import OPTIONAL, PUBLIC
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class CapeVerde(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """Cape Verde holidays.
+
+    References:
+        * [Public holidays in Cape Verde] <https://en.wikipedia.org/wiki/Public_holidays_in_Cape_Verde>
+        * [Today's and Upcoming Holidays in Cape Verde] <https://www.timeanddate.com/holidays/cape-verde/>
+        * [Democracy Day] <https://www.officeholidays.com/holidays/cape-verde/cape-verde-democracy-day>
+        * [National Heroes Day] <https://www.officeholidays.com/holidays/cape-verde/guinea-bissau-national-heroes-day>
+    """
+
+    country = "CV"
+    default_language = "pt_PT"
```

Comment:
```suggestion
    default_language = "pt_CV"
```
It will be more correct. You will also need to make a replacement everywhere and rename the corresponding subdirectory in `locale`.

Also, attributes should be ordered alphabetically.

Reply:
One quick confirmation. If the default language is to be changed to pt_CV then supported languages will be supported_languages = ("en_US", "pt_CV") right?

Reply:
Yes, exactly. pt_PT is replaced with pt_CV everywhere.

Reply:
@KJhellico, you mentioned ordering attributes alphabetically. Is it in the .po file? I am not sure what it means

Reply:
No, I meant class attributes: `country`, `default_language` etc.